jobs/views.py

```diff
@@ -86,3 +87,37 @@ def form_valid(self, form):
         job.company = company
         job.save()
         return super(JobCreate, self).form_valid(form)
+
+
+def create_meetup(request):
```

Comment:
@ksiedlarek why are you not using the generic view as with adding jobs? Is it because of this separate logic for adding the company?